src/Domain/LeanCode.DomainModels.MassTransitRelay/Outbox/SystemTextJsonEventsSerializer.cs

```diff
@@ -1,18 +1,17 @@
-using System;
 using System.Collections.Concurrent;
-using System.Linq;
 using System.Text.Json;
 using LeanCode.Components;
 
 namespace LeanCode.DomainModels.MassTransitRelay.Outbox
 {
     /// <remarks>
-    /// This is not usable yet. `System.Text.Json` does not support private constructors and requires
-    /// users to add `JsonIncludeAttribute` to properties with private setters.
+    /// This is hardly usable. `System.Text.Json` does not support private constructors and requires
```

Comment:
Let's remove the `This is hardly usable` comment, since from now on this will be the only one ;P